docs/4.0/docs/design/datapack.md

```diff
@@ -0,0 +1,91 @@
+---
+sidebar_position: 7
+---
+
+# Datapack Design
+
+Datapack is designed to package data from different CRD with different information granularity.
+
+## How it works
+
+Once a datapack CR is created, datapack controller will base the CR's spec to get information from images and pack it up
+base on different information granularity which is defined at CR's spec type.
+
+And after the CR reconcile process, these information will be placed at the CR's status datas so that you can get the CR
```

Comment:
Explain what‘s the difference of direct get cr?

Reply:
done